storage/types.go

```diff
@@ -282,6 +282,9 @@ type databaseNamespace interface {
 	// Flush flushes in-memory data
 	Flush(blockStart time.Time, flush persist.Flush) error
 
+	// Snapshot snapshots unflushed in-memory data
+	Snapshot(blockStart, callStart time.Time, flush persist.Flush) error
```

Comment:
I'm not sure I understand the question, but callStart is passed in so that all namespace/shard snapshot files can have the same writtenAt time

Reply:
the name gives the impression you were calculating time elapsed using `callStart`. How about renaming all these vars/fields to `SnapshotTime` (instead of callstart/writtenat/etc)

Reply:
done